### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by satishskamath**
* Changed tags `1_cpn_2_nodes` and `1_cpn_4_nodes` to `1cpn_2_nodes` and `1cpn_4_nodes` respectively.
* Described the ESPResSo tests in slight more detail.

### CODE REVIEW

File: docs/test-suite/ReFrame-configuration-file.md [hunk 1]
```diff
@@ -328,9 +328,9 @@ The most common configuration items defined at this level are:
   [`eessi.testsuite.constants.SCALES`](https://github.com/EESSI/test-suite/blob/main/eessi/testsuite/constants.py)
   and define at which scales tests should be run, e.g. single core, half a node, a full node, two nodes, etc. This
   can be used to exclude running at certain scales on systems that would not support it. E.g. some systems might not
-  support requesting multiple partial nodes, which is what the `1_cpn_2_nodes` (1 core per node, on two nodes) and
-  `1_cpn_4_nodes` scales do. One could exclude these by setting e.g.
-  `features: [FEATURES[CPU]] + [s for s in SCALES if s not in ['1_cpn_2_nodes', '1_cpn_4_nodes']]`. With this
+  support requesting multiple partial nodes, which is what the `1cpn_2_nodes` (1 core per node, on two nodes) and
```
Comment (casparvl) [inline docs/test-suite/ReFrame-configuration-file.md:331]: It's actually `1cpn_2nodes`, you'll need to remove one more underscore :)

File: docs/test-suite/usage.md [hunk 2]
```diff
@@ -194,8 +194,8 @@ well as the specific test being carried out.
 | `1_core` | using 1 CPU core 1 GPU |
 | `2_cores` | using 2 CPU cores and 1 GPU |
 | `4_cores` | using 4 CPU cores and 1 GPU |
-| `1_cpn_2_nodes` | using 1 CPU core per node, 1 GPU per node, and 2 nodes |
-| `1_cpn_4_nodes` | using 1 CPU core per node, 1 GPU per node, and 4 nodes |
+| `1cpn_2_nodes` | using 1 CPU core per node, 1 GPU per node, and 2 nodes |
```
Comment (casparvl) [inline docs/test-suite/usage.md:197]: `1cpn_2nodes`

File: docs/test-suite/ReFrame-configuration-file.md (hunk 1 above)
Comment (satishskamath) [reply docs/test-suite/ReFrame-configuration-file.md]: Done!

File: docs/test-suite/usage.md (hunk 2 above)
Comment (satishskamath) [reply docs/test-suite/usage.md]: Done again! :)

**Review by ocaisa — APPROVED** [review]
LGTM

**Pull request merged**